### DISCUSSION TITLE
Back To The Future - EnzoZarx's Theme

### DISCUSSION THREAD

**Discussion by EnzoZarx** [mathoudebine/turing-smart-screen-python#597] [Themes]
### **Hello,**

Here is my contribution to this project _**(Tested on NVIDIA and INTEL | WINDOWS 11)**_.
I have created a theme based on the **DeLorean** Time **Travel** Control Panel from Back To The Future.

**I hope you will enjoy it !**
_(Windows Only)_

**Requirements :**
`pip install WinTmp`

**Setup :** 
1. Change in `theme.yaml` between French and English background
2. Change texts for CPU, GPU, etc... in `theme.yaml`
3. Change the number of hearts for your CPU in `sensor_custom.py`

![preview](https://github.com/user-attachments/assets/80606682-1567-4121-a6e4-52924174b487)

![backgrounden](https://github.com/user-attachments/assets/cbba7ef7-3289-454e-8007-c219c8c6dc34)

[Back To The Future - Theme.zip](https://github.com/user-attachments/files/21203759/Back.To.The.Future.-.Theme.zip)

**Comment by unaigonzalezz** [comment]
I love the idea! Thanks <3

**Reply by EnzoZarx** [reply]
Thanks, I updated it just now with CPU Temps.

**Comment by DJRoby19** [comment]
Tested on linux it doesn't work

**Reply by EnzoZarx** [reply]
Thanks for the information.

**Comment by just1ce-gh** [comment]
Hi! I cant make it work, it doesnt show the stats, any idea?

**Reply by EnzoZarx** [reply]
On what OS are you and also what components do you have ?

**Reply by just1ce-gh** [reply]
Windows 11. I5 14400f, 3070 8gb, 32 ram cordair vengeance, mother gigabyte b760m g p wifi ddr4. Can you give me a litle explanation on how to install it? Maybe im doing something wrong there. Thanks for the reply!

**Reply by EnzoZarx** [reply]
That's really weird, you seems to have the right components compatibility.

Have you done the default install from the official repository ? Did you install the requirements I provided in my presentation ? Did you change all the settings as you want like I mentionned in my post ?

If you need more help let me know.

**Comment by just1ce-gh** [comment]
Ive installed the Windows version from the original repository, installed the last version of python, the pip requirement and I only change the background to english. The backgroung static image works fine, but it doesnt show any sensor info (temps, cpu load, etc).

**Reply by EnzoZarx** [reply]
Do you launch it as an admin or as a user ?

**Reply by just1ce-gh** [reply]
Admin. Dont worry, im gonna try something, then I tell you. Thx!

**Reply by EnzoZarx** [reply]
Yeah let me know if you found something.
